tiledb/sm/rest/rest_client.h

```diff
@@ -175,6 +175,18 @@ class RestClient {
       uint64_t timestamp_end,
       Array* array);
 
+  /**
+   * Gets the array's maximum in memory tile size from the
+   * REST server.
```

Comment:
"REST server." Should fit on the line above.

Reply:
changed